core/shared/src/main/scala/org/http4s/multipart/Part.scala

```diff
@@ -44,7 +43,7 @@ object Part {
   def formData(name: String, value: String, headers: Header.ToRaw*): Part[fs2.Pure] =
     Part(
       Headers(`Content-Disposition`("form-data", Map(ci"name" -> name))).put(headers: _*),
-      Entity.Strict(Chunk.array(value.getBytes(UTF_8))),
+      Entity.Strict(ByteVector.encodeUtf8(value).toOption.get),
```

Comment:
To work around `.toOption.get` we can do this probably
```suggestion
      Entity.Strict(ByteVector.view(value.getBytes(UTF_8))),
```

Reply:
@danicheg thanks, I'm pretty curious what the difference is, so some stupid questions: is `value.getBytes(UTF_8)` guaranteed to never throw an error, why is that? What makes it different from `ByteVector.encodeUtf8`? Should this encoder be `Pure` if there can be error?

Reply:
Honestly, I'm not a `scodec` expert, I can just suppose things. 
Speaking from the perspective of the types, we already had the version on `value.getBytes(UTF_8)` and didn't check possible exceptions. Though, I think that `scodec` is much safer if we wouldn't lose on the purpose of this safety through `.toOption.get`. 
Basically, your variant is as safe as the earlier one, I guess. Just users might face a "None.get" exception, instead of what Java API could throw. Will that exception be more clear to users? I dunno.

Reply:
> Basically, your variant is as safe as the earlier one

Thanks, yes, exactly. And the error message is a regression :) since this is 1.0 and we can be breaking things, maybe this should not be `Pure` then.

Reply:
Ah, `getBytes` is total and never throws an error.
> This method always replaces malformed-input and unmappable-character sequences with this charset's default replacement byte array.